treeherder/perf/alerts.py

```diff
@@ -83,6 +96,37 @@ def generate_new_alerts_in_series(signature):
                     prev_value, new_value, signature.lower_is_better
                 )
 
+                noise_profile = "N/A", 0
+                try:
+                    # Gather all data up to the current data point that
+                    # shows the regression and obtain a noise profile on it.
+                    # This helps us to ignore this alert and others in the
+                    # calculation that could influence the profile.
+                    noise_data = []
+                    for point in analyzed_series:
+                        if point == cur:
+                            break
+                        noise_data.append(geomean(point.values))
+
+                    noise_profile = deviance(noise_data)
```

Comment:
I've noticed that `noise_profile[1]` is not used. Instead of `noise_profile = deviance(noise_data)` you could do `noise_profile, _ = deviance(noise_data)` and then use `noise_profile` instead of `noise_profile[0]` where needed. Hope it makes sense.